### ISSUE TITLE
Cross-staff beams

### ISSUE THREAD

**Issue by craigsapp** [rism-ch/verovio#912]
Is it currently possible to move a beam to the next higher/lower staff:

<img width="299" alt="screen shot 2018-07-04 at 2 40 58 pm" src="https://user-images.githubusercontent.com/3487289/42277664-6fd65850-7f98-11e8-8394-4a499886d07b.png">

If not then would `beam@beam.with="below"` be the command to move it down one staff?

MEI test data:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="http://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="http://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="4.0.0">
    <meiHead>
        <fileDesc>
            <titleStmt>
                <title />
            </titleStmt>
            <pubStmt />
        </fileDesc>
        <encodingDesc>
            <appInfo>
                <application isodate="2018-07-04T14:36:55" version="2.0.0-dev-f0e1012">
                    <name>Verovio</name>
                    <p>Transcoded from Humdrum</p>
                </application>
            </appInfo>
        </encodingDesc>
        <workDesc>
            <work>
                <titleStmt>
                    <title />
                </titleStmt>
            </work>
        </workDesc>
        <extMeta>
            <frames xmlns:humxml="http://www.humdrum.org/ns/humxml">
                <metaFrame n="11" token="!!!RDF**kern: &lt; = below" xml:id="loc11">
                    <frameInfo>
                        <startTime float="2" />
                        <frameType>reference</frameType>
                        <referenceKey>RDF**kern</referenceKey>
                        <referenceValue>&lt; = below</referenceValue>
                    </frameInfo>
                </metaFrame>
                <metaFrame n="12" token="!!!RDF**kern: &gt;  = above" xml:id="loc12">
                    <frameInfo>
                        <startTime float="2" />
                        <frameType>reference</frameType>
                        <referenceKey>RDF**kern</referenceKey>
                        <referenceValue>&gt;  = above</referenceValue>
                    </frameInfo>
                </metaFrame>
            </frames>
        </extMeta>
    </meiHead>
    <music>
        <body>
            <mdiv xml:id="mdiv-0000001289723225">
                <score xml:id="score-0000001288769168">
                    <scoreDef xml:id="scoredef-0000001254256541">
                        <staffGrp xml:id="staffgrp-0000000670102848" symbol="brace" barthru="true">
                            <staffDef xml:id="staffdef-0000000681240754" clef.shape="G" clef.line="2" meter.count="4" meter.unit="4" n="1" lines="5">
                                <label xml:id="label-0000001072277403" />
                            </staffDef>
                            <staffDef xml:id="staffdef-0000000507121344" clef.shape="F" clef.line="4" meter.count="4" meter.unit="4" n="2" lines="5" />
                        </staffGrp>
                    </scoreDef>
                    <section xml:id="section-L1F1">
                        <measure xml:id="measure-L4" right="end" n="1">
                            <staff xml:id="staff-L4F2N1" n="1">
                                <layer xml:id="layer-L4F2N1" n="1">
                                    <beam xml:id="beam-L5F2-L8F2">
                                        <note xml:id="note-L5F2" dur="16" staff="2" oct="3" pname="c" accid.ges="n" />
                                        <note xml:id="note-L6F2" dur="16" staff="2" oct="3" pname="b" accid.ges="n" />
                                        <note xml:id="note-L7F2" dur="16" staff="2" oct="3" pname="d" accid.ges="n" />
                                        <note xml:id="note-L8F2" dur="16" staff="2" oct="3" pname="c" accid.ges="n" />
                                    </beam>
                                    <note xml:id="note-L9F2" dur="4" staff="2" oct="3" pname="f" stem.dir="up" accid.ges="n" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-L4F1N1" n="2">
                                <layer xml:id="layer-L4F1N1" n="1">
                                    <note xml:id="note-L5F1" dur="4" oct="2" pname="c" stem.dir="down" accid.ges="n" />
                                    <note xml:id="note-L9F1" dur="4" oct="2" pname="b" stem.dir="down" accid.ges="n" />
                                </layer>
                            </staff>
                        </measure>
                    </section>
                </score>
            </mdiv>
        </body>
    </music>
</mei>
```

**Comment by lpugin** [comment]
Implemented here 28b88520f7191a9fff5114ec08d989f2237ff99f

**Comment by craigsapp** [comment]
Looks great:

<img width="398" alt="screen shot 2018-07-25 at 12 47 27 am" src="https://user-images.githubusercontent.com/3487289/43170144-2cb7ef36-8fa4-11e8-8a9b-583be94909fa.png">


```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="http://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="http://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="4.0.0">
    <meiHead>
        <fileDesc>
            <titleStmt>
                <title />
            </titleStmt>
            <pubStmt />
        </fileDesc>
        <encodingDesc>
            <appInfo>
                <application isodate="2018-07-04T14:36:55" version="2.0.0-dev-f0e1012">
                    <name>Verovio</name>
                    <p>Transcoded from Humdrum</p>
                </application>
            </appInfo>
        </encodingDesc>
        <workDesc>
            <work>
                <titleStmt>
                    <title />
                </titleStmt>
            </work>
        </workDesc>
        <extMeta>
            <frames xmlns:humxml="http://www.humdrum.org/ns/humxml">
                <metaFrame n="11" token="!!!RDF**kern: &lt; = below" xml:id="loc11">
                    <frameInfo>
                        <startTime float="2" />
                        <frameType>reference</frameType>
                        <referenceKey>RDF**kern</referenceKey>
                        <referenceValue>&lt; = below</referenceValue>
                    </frameInfo>
                </metaFrame>
                <metaFrame n="12" token="!!!RDF**kern: &gt;  = above" xml:id="loc12">
                    <frameInfo>
                        <startTime float="2" />
                        <frameType>reference</frameType>
                        <referenceKey>RDF**kern</referenceKey>
                        <referenceValue>&gt;  = above</referenceValue>
                    </frameInfo>
                </metaFrame>
            </frames>
        </extMeta>
    </meiHead>
    <music>
        <body>
            <mdiv xml:id="mdiv-0000001289723225">
                <score xml:id="score-0000001288769168">
                    <scoreDef xml:id="scoredef-0000001254256541">
                        <staffGrp xml:id="staffgrp-0000000670102848" symbol="brace" barthru="true">
                            <staffDef xml:id="staffdef-0000000681240754" clef.shape="G" clef.line="2" meter.count="4" meter.unit="4" n="1" lines="5">
                                <label xml:id="label-0000001072277403" />
                            </staffDef>
                            <staffDef xml:id="staffdef-0000000507121344" clef.shape="F" clef.line="4" meter.count="4" meter.unit="4" n="2" lines="5" />
                        </staffGrp>
                    </scoreDef>
                    <section xml:id="section-L1F1">
                        <measure xml:id="measure-L4" right="end" n="1">
                            <staff xml:id="staff-L4F2N1" n="1">
                                <layer xml:id="layer-L4F2N1" n="1">
                                    <beam staff="2" beam.with="below" xml:id="beam-L5F2-L8F2">
                                        <note xml:id="note-L5F2" dur="16" staff="2" oct="3" pname="c" accid.ges="n" />
                                        <note xml:id="note-L6F2" dur="16" staff="2" oct="3" pname="b" accid.ges="n" />
                                        <note xml:id="note-L7F2" dur="16" staff="2" oct="3" pname="d" accid.ges="n" />
                                        <note xml:id="note-L8F2" dur="16" staff="2" oct="3" pname="c" accid.ges="n" />
                                    </beam>
                                    <note xml:id="note-L9F2" dur="4" staff="2" oct="3" pname="f" stem.dir="up" accid.ges="n" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-L4F1N1" n="2">
                                <layer xml:id="layer-L4F1N1" n="1">
                                    <note xml:id="note-L5F1" dur="4" oct="2" pname="c" stem.dir="down" accid.ges="n" />
                                    <note xml:id="note-L9F1" dur="4" oct="2" pname="b" stem.dir="down" accid.ges="n" />
                                </layer>
                            </staff>
                        </measure>
                    </section>
                </score>
            </mdiv>
        </body>
    </music>
</mei>
```
